packages/hdwallet-vultisig/src/vultisig.ts

```diff
@@ -0,0 +1,534 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+import { Address } from "@shapeshiftoss/hdwallet-core";
+import Base64 from "base64-js";
+import * as bitcoinMsg from "bitcoinjs-message";
+import { keccak256, recoverAddress } from "ethers/lib/utils.js";
+import isObject from "lodash/isObject";
+
+import * as btc from "./bitcoin";
+import * as cosmos from "./cosmos";
+import * as eth from "./ethereum";
+import { solanaSendTx, solanaSignTx } from "./solana";
+import * as thorchain from "./thorchain";
+import { VultisigBftProvider, VultisigEvmProvider, VultisigSolanaProvider, VultisigUtxoProvider } from "./types";
+
+export function isVultisig(wallet: core.HDWallet): wallet is VultisigHDWallet {
+  return isObject(wallet) && (wallet as any)._isVultisig;
+}
+
+export class VultisigHDWalletInfo
+  implements
+    core.HDWalletInfo,
+    core.BTCWalletInfo,
+    core.ETHWalletInfo,
+    core.SolanaWalletInfo,
+    core.ThorchainWalletInfo,
+    core.CosmosWalletInfo
+{
+  readonly _supportsBTCInfo = true;
+  readonly _supportsETHInfo = true;
+  readonly _supportsSolanaInfo = true;
+  readonly _supportsThorchainInfo = true;
+  readonly _supportsCosmosInfo = true;
+
+  evmProvider: VultisigEvmProvider;
```

Comment:
Think we don't need provider bits for info class

Reply:
We aren't using the `emvProvider` for anything here so we can remove.